cmd/server-main.go

```diff
@@ -138,7 +138,12 @@ func initConfig() {
 			console.Fatalf("Invalid value ‘%s’ in MINIO_BROWSER environment variable.", browser)
 		}
 
-		globalIsEnvBrowser = strings.EqualFold(browser, "off")
+		// browser Envs are set globally.
+		globalIsEnvBrowser = true
+	} else {
+		// browser envs are not through env,
+		// but by default browser is on nevertheless.
+		browser = "on"
```

Comment:
Not clear why this is needed, besides, we are setting this `browser` variable to `envParams`, I think this could lead to a more confusion. envParams should only contain exact env variables.